AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.m

```diff
@@ -32,7 +32,18 @@ + (void)performDbTokenAsyncOperationWithHttpClient:(id<MSHttpClientProtocol>)htt
                                          partition:(NSString *)partition
                                includeExpiredToken:(BOOL)includeExpiredToken
                                  completionHandler:(MSGetTokenAsyncCompletionHandler)completionHandler {
-
+    if (![MSTokenExchange isValidPartitionName:partition]) {
+        MSLogError([MSDataStore logTag], @"Request invalid partition %@", partition);
```

Comment:
The "Request invalid partition %@" error message can be improved, something like "Can't perform token exchange because partition name %@ is invalid."

(Please add a dot `.` to the end of the log sentence.) 😄